Android/app/src/main/res/layout/activity_my_trip_info.xml

```diff
@@ -1,147 +1,159 @@
 <?xml version="1.0" encoding="utf-8"?>
-<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+<android.support.design.widget.CoordinatorLayout
```

Comment:
The changes in this file should also not be a part of this PR

Reply:
Snackbars work best if they are displayed inside of a CoordinatorLayout so I added it and the ID to get the view.
TravelmateSnackbars.createSnackBar(findViewById(R.id.activityAddNewTrip),
R.string.trip_added, Snackbar.LENGTH_LONG).show();